### PR TITLE
129 low results removes the filter button

### PR DESCRIPTION
**Pull request by jthistle**
https://trello.com/c/NvZgibXu/129-low-results-removes-the-filter-button

This just always displays the navigation, which should be fine, really.

### CODE REVIEW

**Review by Nikomus — COMMENTED** [review]
We still have the switch on `filteredServices.length > 0` of whether to display the empty state or not, so we still lose the filer when there are 0 results. Are you able to fix that here too?

**Comment by jthistle** [comment]
@Nikomus oops, I forgot about that case. Will fix.

**Review by Nikomus — APPROVED** [review]
All looks very reasonable ✔️

File: src/Components/EmptyList.tsx [hunk 1]
```diff
@@ -26,7 +34,11 @@ export const EmptyList = ({ listType }: EmptyListProps): JSX.Element => {
       content = (
         <span>
           We weren&apos;t able to find any services matching your current
-          filters. Try expanding your filters to show more results.
+          filters. Try{' '}
+          <Link href="/filters" passHref>
```
Comment (Nikomus) [inline src/Components/EmptyList.tsx:38]:
```suggestion
          <Link href="/filter" passHref>
```

**Pull request merged**